HunterJuneau/dine-and-dash
# Change to Add ProductOrders API Call and created API Call to Get All ProductOrders by Order Id

## Description
Change to Add ProductOrders API Call and created API Call to Get All ProductOrders by Order Id

## Motivation and Context
Needed this to for our Product Orders to reflect an accurate total cost and needed All product Orders by Order Id for the User Detailed Orders View

## How Can This Be Tested?
First, head to the design of the Product Orders Table in SQL. 
Added a Column to this Table named 'totalCost'. Set it to a decimal and allow nulls.
Then add a product order like you would before, but now you will see that the total cost of the this product order is the price of the item * quantity order. This does not update already made product orders total cost, only new ones that we add. 

To test the Product Orders by Order Id Api Call, simply grab an orderId and add it to the call. 
You should see all the Product Orders associated with that Order and all of it's details. 


## Types of changes
